src/ConcordiumNetSdk/Transactions/PreparedAccountTransaction.cs

```diff
@@ -0,0 +1,61 @@
+using ConcordiumNetSdk.Types;
+
+namespace ConcordiumNetSdk.Transactions;
+
+/// <summary>
+/// Represents an account transaction which is ready to be signed.
+///
+/// A prepared account transaction is <see cref="T"/>
```

Comment:
Is there something off in this sentence? This class is defining the Prepared transaction.